examples/vqe_noisyopt.py

```diff
@@ -0,0 +1,194 @@
+"""
+VQE with finite measurement shot noise
+"""
+
+from functools import partial
+import numpy as np
+from scipy import optimize
+import optax
+import tensorcircuit as tc
+from tensorcircuit import experimental as E
+from noisyopt import minimizeCompass, minimizeSPSA
+from tabulate import tabulate
+
+seed = 42
+np.random.seed(seed)
```

Comment:
not sure whether this is necessary, as the randomness used in this script is explicitly managed by jax random keys below